src/workspace/shared/StatusLayoutButton.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
This is also present in the /src/shared directory.

Reply:
Updated.